tuna/import_benchmark.py

```diff
@@ -25,33 +25,78 @@
+import os
 from sqlalchemy.exc import IntegrityError
+from sqlalchemy.orm.exc import NoResultFound
 
 from tuna.dbBase.sql_alchemy import DbSession
 from tuna.parse_args import TunaArgs, setup_arg_parser
 from tuna.utils.logger import setup_logger
 from tuna.miopen.benchmark import Framework, Model, FrameworkEnum, ModelEnum
+from tuna.miopen.tables import MIOpenDBTables
+from tuna.config_type import ConfigType
+from tuna.driver_conv import DriverConvolution
+from tuna.driver_bn import DriverBatchNorm
 
 LOGGER = setup_logger('import_benchmarks')
 
 
 def parse_args():
   """Parsing arguments"""
   parser = setup_arg_parser('Import benchmark performance related items',
-                            [TunaArgs.VERSION])
-  parser.add_argument('--update_framework',
+                            [TunaArgs.CONFIG_TYPE])
+  group1 = parser.add_mutually_exclusive_group()
+  group2 = parser.add_mutually_exclusive_group()
+  group3 = parser.add_mutually_exclusive_group()
+  group1.add_argument('--update_framework',
                       action="store_true",
                       dest='update_framework',
                       help='Populate framework table with all framework enums')
-  parser.add_argument('--add_model',
+  group2.add_argument('--add_model',
                       dest='add_model',
                       type=ModelEnum,
                       choices=ModelEnum,
                       help='Populate table with new model and version')
-  parser.add_argument('--print_models',
+  group3.add_argument('--print_models',
                       dest='print_models',
                       action='store_true',
                       help='Print models from table')
+  parser.add_argument('--add_benchmark',
+                      dest='add_benchmark',
+                      action='store_true',
+                      help='Insert new benchmark')
+  parser.add_argument('-m',
+                      '--model',
+                      dest='model',
+                      type=ModelEnum,
+                      choices=ModelEnum,
+                      required=False,
+                      help='Specify model')
+  parser.add_argument('-F',
+                      '--framework',
+                      dest='framework',
+                      type=FrameworkEnum,
+                      choices=FrameworkEnum,
+                      help='Specify framework.')
+  parser.add_argument('-d',
+                      '--driver',
+                      dest='driver',
+                      type=str,
+                      default=None,
+                      required=False,
+                      help='Specify driver cmd')
+  parser.add_argument('-g',
+                      '--gpu_count',
+                      dest='gpu_count',
+                      type=int,
+                      default=None,
+                      required=False,
+                      help='Specify number of gpus the benchmark runs on')
+  parser.add_argument('-f',
```

Comment:
Is `batchsize` missing ?

Reply:
Oh good catch, i must have deleted it. Re-adding